### PR TITLE
Add una to list of MPR helpers

### LINKED ISSUE #15: Add Una MPR helper

### PR DESCRIPTION
**Pull request by RudraSwat**
Closes #15

### CODE REVIEW

**Review by gamer4life1 — APPROVED** [review]
LGTM

File: docs/mpr/list-of-mpr-helpers.md [hunk 1]
```diff
@@ -23,4 +23,5 @@ Outside of that listed above:
 ## Helpers
 | Name  | MPR Package                                             | Homepage                                                                                 | Programming Language | Search | Clone   | Install | Update | APT-Install | APT-Update | Git-Update               | MPR-Dependencies  |
 | :---  | :------------------------------------------------------ | :--------------------------------------------------------------------------------------- | :------------------- | :----- | :----   | :------ | :----- | :---------- | :--------- | :----------------------- | :---------------  |
+| una   | [una-bin](https://{{env.mpr_url}}/packages/una-bin)             | [https://{{env.github_url}}/AFK-OS/una](https://{{env.github_url}}/AFK-OS/una) | Bash               | Yes    | Yes | Yes     | Yes    | Yes          | Yes         | Yes                       | Yes                |
```
Comment (hwittenborn) [inline docs/mpr/list-of-mpr-helpers.md:26]: Can you space out the `|` characters to align with those for Tap so everything looks neater?

Not that I'm trying to prioritize my helper, but I planned on keeping any helpers listed here in alphabetical order, so una would probably go below Tap here :p

I also remember you capitalizing `una` as `Una` in the makedeb support rooms a few times - I don't care which way it's done, I just wanted to confirm that you wanted it written as `una`.

**Review by hwittenborn — CHANGES_REQUESTED** [review]

File: docs/mpr/list-of-mpr-helpers.md (hunk 1 above)
Comment (RudraSwat) [reply docs/mpr/list-of-mpr-helpers.md]: Sure, I've aligned the pipe characters for una with those for Tap. I've also put Tap above una now.

> I also remember you capitalizing `una` as `Una` in the makedeb support rooms a few times - I don't care which way it's done, I just wanted to confirm that you wanted it written as `una`.

I'm alright with both, but prefer `una` above `Una`.

**Review by hwittenborn — APPROVED** [review]
Let's get this merged! :D

**Pull request merged**